tests/test_configs/config.yaml

```diff
@@ -0,0 +1,153 @@
+imaging_specs:
+  local_storage_directory: .
+  external_storage_directory: .
+  subject_id: sim_test
+  tile_prefix: tile
+  tile_overlap_x_percent: 15
+  tile_overlap_y_percent: 10
+  z_step_size_um: 1
+  volume_x_um: 10615.616
+  volume_y_um: 7958.7
+  volume_z_um: 100
+  laser_wavelengths:
+  - 561  # test comment!
+         # test multi-line comment!
+experiment_specs:
+  experimenters_name: Sim Robot
+  immersion_medium: 0.05X SSC
+  immersion_medium_refractive_index: 1.33
+  x_anatomical_direction: Posterior_to_anterior
+  y_anatomical_direction: Inferior_to_superior
+  z_anatomical_direction: Right_to_left
+design_specs:
+  slit_width_pixels: 489.0
+waveform_specs:
+  ttl_pulse_time_s: 0.01
+  frame_rest_time_s: 0.05
+compressor_specs:
+  image_stack_chunk_size: 64
+  compressor_thread_count: 32
+  compression_style: lz4
+file_transfer_specs:
+  protocol: xcopy
+  protocol_flags: /j/i/y
+camera_specs:
+  egrabber_frame_buffer: 8
+  unpacking_mode: Msb
+  digital_gain_adu: 1
+  line_interval_us: 20.44
+tile_specs:
+  x_field_of_view_um: 10615.616
+  y_field_of_view_um: 7958.72
+  row_count_pixels: 10640
+  column_count_pixels: 14192
+  data_type: uint16
+sample_stage_specs:
+  backlash_reset_distance_um: 4
+daq_driver_kwds:
+  dev_name: Dev2
+  samples_per_sec: 10000.0
+  ao_channels:
+    etl: 0
+    camera: 1
+    stage: 2
+    '488': 3
+    '638': 4
+    '561': 13
+    '405': 6
+    galvo_a: 17
+    galvo_b: 19
+tiger_controller_driver_kwds:
+  com_port: COM3
+estimates: {}
+channel_specs:
+  '488':
```

Comment:
Similarly here, quotes are needed to explicitly say that 488 is a string and not a number?

Reply:
Totally. 
YAML format "autodetects" simple types. That is to say, a list of integers would be interpretted as a list of integers unless you explicitly say that they're strings. In our old TOML config, the laser channel specs keys were actually in string format, so this is preserved here.